dashboard/README.md

```diff
@@ -19,7 +19,7 @@ The Dashboard is a SPA(Single Page App) made with React and will require the fol
 This may be simpler than deploying the builder and connecting your OpenFaaS Cloud to a GitHub App.
 
 ```
-$ faas-cli store deploy figlet --name alexellis-figlet --label Git-Owner=alexellis \
+$ faas-cli store deploy figlet --name alexellis-figlet --label com.openfaas.cloud.git-owner=alexellis \
  --label com.openfaas.cloud.git-repo=figlet \
  --label com.openfaas.cloud.git-sha=665d9597547d8e0425630ba2dbb73c2951a61ce2 \
  --label com.openfaas.cloud.git-deploytime=1533026741 \
```

Comment:
Should we add a note in this file saying that users need to have public visibility in their orgs to show up?

Maybe attach a very small screenshot, the one I sent you showing how to change it?

Reply:
Is this the best place for that? Is it more general openfaas-cloud thing rather than the dashboard readme?

Reply:
I'll let you decide. I thought it might be OK here.